backend/import_status/get.py

```diff
@@ -1,39 +1,22 @@
 """Import Status handler function."""
 from http import HTTPStatus
-from json import dumps, loads
-from typing import TYPE_CHECKING
+from json import dumps
 
-import boto3
 from jsonschema import ValidationError, validate
 
-from ..api_keys import SUCCESS_KEY
+from ..api_keys import EVENT_KEY
 from ..api_responses import error_response, success_response
 from ..error_response_keys import ERROR_KEY
 from ..log import set_up_logging
-from ..step_function import get_tasks_status
-from ..step_function_keys import (
-    DATASET_ID_KEY,
-    EXECUTION_ARN_KEY,
-    IMPORT_DATASET_KEY,
-    STEP_FUNCTION_KEY,
-    VALIDATION_KEY,
-    VERSION_ID_KEY,
-)
+from ..step_function import get_import_status_given_arn
+from ..step_function_keys import EXECUTION_ARN_KEY
 from ..types import JsonObject
 
-if TYPE_CHECKING:
-    # When type checking we want to use the third party package's stub
-    from mypy_boto3_stepfunctions import SFNClient
-else:
-    # In production we want to avoid depending on a package which has no runtime impact
-    SFNClient = object
-
-STEP_FUNCTIONS_CLIENT: SFNClient = boto3.client("stepfunctions")
 LOGGER = set_up_logging(__name__)
 
 
 def get_import_status(body: JsonObject) -> JsonObject:
-    LOGGER.debug(dumps({"event": body}))
```

Comment:
Good catch, but does this maybe belong in a separate PR? There are a bunch more occurrences of this, according to `git grep '"event"'`.

Reply:
will follow up with another PR